### DISCUSSION TITLE
[UX] The website shows "V5/V6" by default. I thought I was on docs for V5/V6.

### DISCUSSION THREAD

**Discussion by ThePiyushAggarwal** [react-hook-form/react-hook-form#9961] [General]
And when I clicked on it, I was taken to a website with legacy written in URL and "Version 7" written in the button.

This was misleading for me at least. It could be changed into a dropdown. I think by default the version shown up there should be the version that we are looking at.

**Comment by Moshyfawn** [comment]
Makes sense.

I've started some docs improvement work but I'm yet to finish it. Really hoping to push out at least a partial update soon 🤞
